minimal/modules/dnpm-node-compose.yml

```diff
@@ -74,9 +74,27 @@ services:
         condition: service_healthy
     labels:
       - "traefik.enable=true"
-      - "traefik.http.routers.dnpm-backend.rule=PathPrefix(`/api`)"
       - "traefik.http.services.dnpm-backend.loadbalancer.server.port=9000"
+      # expose everything
+      - "traefik.http.routers.dnpm-backend.rule=PathPrefix(`/api`)"
       - "traefik.http.routers.dnpm-backend.tls=true"
+      - "traefik.http.routers.dnpm-backend.service=dnpm-backend"
```

Comment:
Do we even need this router anymore I think the other 2 cover all the relevant cases?

Reply:
I think we still need this for the frontend to access the data.